src/_P076_HLW8012.ino

```diff
@@ -104,11 +137,41 @@ boolean Plugin_076(byte function, struct EventStruct *event, String &string) {
     byte cf_trigger  = Settings.TaskDevicePluginConfig[event->TaskIndex][5];
     byte cf1_trigger = Settings.TaskDevicePluginConfig[event->TaskIndex][6];
 
+    byte devicePinSettings =
+                          Settings.TaskDevicePluginConfig[event->TaskIndex][7];
+
+
+
+    String predefinedNames[p076_PinSettingsCount];
+    predefinedNames[0] = p076_PredefinedPinSettings[0].Device_Name;
+    predefinedNames[1] = p076_PredefinedPinSettings[1].Device_Name;
+    predefinedNames[2] = p076_PredefinedPinSettings[2].Device_Name;
+    predefinedNames[3] = p076_PredefinedPinSettings[3].Device_Name;
+    predefinedNames[4] = p076_PredefinedPinSettings[4].Device_Name;
+    predefinedNames[5] = p076_PredefinedPinSettings[5].Device_Name;
+    predefinedNames[6] = p076_PredefinedPinSettings[6].Device_Name;
+    predefinedNames[7] = p076_PredefinedPinSettings[7].Device_Name;
+    predefinedNames[8] = p076_PredefinedPinSettings[8].Device_Name;
+    predefinedNames[9] = p076_PredefinedPinSettings[9].Device_Name;
```

Comment:
Why not use a loop to fill the `predefinedNames` ?
Both indices are the same and sequential.

Reply:
good idea, will be fiexed